server/src/main/java/server/Server.java

```diff
@@ -0,0 +1,43 @@
+package server;
+
+import model.Car;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import servise.CarService;
+
+import java.io.IOException;
+import java.io.ObjectInputStream;
+import java.io.ObjectOutputStream;
+import java.net.ServerSocket;
+import java.net.Socket;
+
+public class Server {
+
+    private final static Logger logger = LoggerFactory.getLogger(Server.class);
+    public static final int port = 4444;
+    private static CarService carService = new CarService();
```

Comment:
why do you need CarService as static variable?

Reply:
it's from previous version when I've tried start Server from another controller